annotations/fenix/metrics/recent_tabs.recent_tab_opened/README.md

```diff
@@ -0,0 +1,8 @@
+---
+tags:
+  - RecentTabs
+---
+
+There are two types of tabs that show up in the recent tabs section on the Fenix home screen:
+tabs that have media in progress and recently used tabs.
+This metric is used to collect info about how many users are opening a tab they have recently used.
```

Comment:
Since you're the developer, I'd say you could probably add this type of context to the metric description itself. That said, I see no problem adding this here (it's almost certainly less hassle!).